tests/providers/presto/transfers/test_presto_to_slack.py

```diff
@@ -41,37 +41,60 @@ def _construct_operator(**kwargs):
         operator = PrestoToSlackOperator(task_id=TEST_DAG_ID, **kwargs)
         return operator
 
-    @mock.patch('airflow.providers.presto.transfers.presto_to_slack.PrestoHook')
-    @mock.patch('airflow.providers.presto.transfers.presto_to_slack.SlackWebhookHook')
-    def test_hooks_and_rendering(self, mock_slack_hook_class, mock_presto_hook_class):
+    @mock.patch('airflow.providers.slack.transfers.sql_to_slack.SlackWebhookHook')
+    def test_hooks_and_rendering_with_slack_conn(self, mock_slack_hook_class):
```

Comment:
This test file should be minimal now.. only the tests that we need to verify that `PrestoToSlackOperator` is successfully "converted" into the new operator. All the other tests logic should be in `test_sql_to_slack.py`
The reasoning for this is that once we remove `PrestoToSlackOperator` we will also remove it's test file and by doing so we don't want to lose any tests that are important.

Reply:
I left the test as is, but changed the mocking location.
now, the test is verifying that the message that is getting built using the operator is actually triggered from the generic `sql_to_slack.SlackWebhookHook`.
All the other tests like rendering different df names and such are in `SqlToSlackOperator` tests.. So I think that the tests are pretty minimal as is, and we can delete them and you won't lose any "important" tests, or you will miss some logic..
My guideline here was end to end test `PrestoToSlackOperator`, and make sure that the actual call is called from `SqlToSlackOperator`.. if we remove `PrestoToSlackOperator` eventually, I see no issue here, as e2e is not important anymore :)

Or am I mistaken here?

Reply:
fair point!